service.yml

```diff
@@ -0,0 +1,22 @@
+name: kafka-streams-examples
+lang: java
+lang_version: 8
+git:
+  enable: true
+codeowners:
+  enable: true
+semaphore:
+  enable: true
+  pipeline_type: cp-dockerfile
```

Comment:
the value of pipeline should `pipeline_type: <cc_mk_include | bazel | cp>`
https://confluentinc.atlassian.net/wiki/spaces/TOOLS/pages/1342058645/Semaphore+FAQ#:~:text=pipeline_type%3A%20%3Ccc_mk_include%20%7C%20bazel%20%7C%20cp%3E

Reply:
I got that from https://confluentinc.atlassian.net/wiki/spaces/TOOLS/pages/3211693075/Jenkins+-+Semaphore+self+migration+guide#How-do-I-migrate-dockerfile?